docs/handbook/strategy.md

```diff
@@ -66,6 +66,67 @@ What we will build:
 
 Making money:
 
-* We will focus on small customers first. This is because we will optimize for learning. There are more of them and they will be faster to learn from. Product analytics tools often charge 100 person startups $20-70K/year.
-* We can undercut SAAS product analytics tools because we have no hosting costs.
-* Over time, we will put more work into our innovation, and the biggest and brightest companies will realize that we are the market leader.
+We'll have three routes to generating revenue
+
+* Set up a self-serve hosted version
+* Build some of the growth/ultimate plan features like experimentation
+* Support self-hosted
+
+## Hiring
+
+Now that we've raised our seed, we have the money to comfortably hire about 8 remote people and have 3 years of runway, taking us to the end of Q2 2023.
+
+A couple of "rules":
+* Tim and James would like to remain [working managers](https://m.signalvnoise.com/this-is-why-we-have-working-managers-at-basecamp-and-why-microsoft-and-apple-stumbled-when-they/).
+  * That means hiring [managers of one](https://signalvnoise.com/posts/1430-hire-managers-of-one).
+* We are too small for tiered management, and will be until after we raise a Series A.
+* We don't _have_ to hire all the people we have money for. In fact, we should find ways of achieving our roadmap with the fewest people possible.
+
+At the moment, our team is heavily weighted towards frontend/backend development. We're currently lacking:
+* Deep experience with DevOps and the various cloud providers
+  * We need this because we want to be able to deploy PostHog on as many platforms as possible + support
+* Experience scaling systems to 100s of requests/sec
+* Experience scaling Postgres to terabytes
+* Great visual/UX designer
+* Marketing/growth
+
+We could also use more firepower to deliver features faster, though we should be careful with hiring more generalists early on. We could end up needing too much co-ordination. The priority should be a devops/backend scaling generalist.
+
+### The plan:
+* 2020 Q2: Hire 1 engineer, total of 4 engineers (including Tim)
+* 2020 Q3: Hire 1 designer, total 4 engineers 1 designer
```

Comment:
Afterthought: Some of these feelings come from reading about Notion's meteoric success and their composition is somewhat unintuitive because at 30 people as of December 2019, 10 of them are customer support, 10 are engineers, and I'm not sure what the rest of the 10 are haha but I think the distinct emphasis on customer and community has helped them understand their users much better than most competitors in the space

Reply:
This is a great point re customer support.

The downside of bringing customer support in too early is that it'll silo engineering from users. Our customers are particularly technical too (much moreso than Notion) so it will require an engineer in many cases early on to do this properly.

I think for us customer success will have two phases:

1 - community. It's probably best to do 'everyone is a moderator' for now, so as an engineer you are interacting with our users. That'll help you work out if the new stuff you're building is any good / difficult to use / etc.  If this work becomes overwhelming so the roadmap feature work slows down, we could just get another full stack person so we have more bandwidth overall and put in a rotation of who looks after issues being raised.
2 - paid. Once we have paying customers we need to nail this. The first one should be heavily looked after by our engineers. I'd suggest we review by the time the second customer joins? Splunk apparently did a great job in our space of this - they focussed customer support on helping their users actually achieve the answers to questions they had to answer with their analytics, not just on "do you know all the features".

Reply:
100% agree with this. 'support' should be responsibility of engineers for a long time. I'm getting so much valuable feedback and insight from doing customer support, and people are super happy when I fix whatever bug they have in 30 minutes. That could never happen if that person had to go through a CS person first, who had to escalate to an engineer.

Reply:
Added this more explicitely